azurerm/internal/services/network/web_application_firewall_policy_resource.go

```diff
@@ -282,6 +282,26 @@ func resourceWebApplicationFirewallPolicy() *schema.Resource {
 				},
 			},
 
+			"http_listener_ids": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					Required:     true,
+					ValidateFunc: validate.ApplicationGatewayHTTPListenerID,
+				},
+			},
+
+			"path_based_rule_ids": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					Required:     true,
+					ValidateFunc: validate.ApplicationGatewayPathBasedRuleID,
```

Comment:
I assume this wants to become `ApplicationGatewayURLPathMapPathRule`.

Reply:
Thanks, I was a little too quick on the trigger there 😅